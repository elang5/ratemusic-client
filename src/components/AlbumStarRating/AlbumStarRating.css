.AlbumStarRating {
  display: inherit;
  padding-top: 1em;
  color: rgb(252, 60, 57);
}