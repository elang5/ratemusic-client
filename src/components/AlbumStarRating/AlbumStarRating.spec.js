import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import AlbumStarRating from './AlbumStarRating'

describe(`AlbumStarRating Component`, () => {

  it(`renders the AlbumStarRating component`, () => {
    const wrapper = shallow(<AlbumStarRating />)
    expect(toJson(wrapper)).toMatchSnapshot()
  })
})