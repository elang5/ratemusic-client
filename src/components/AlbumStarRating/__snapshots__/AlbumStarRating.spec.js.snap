// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AlbumStarRating Component renders the AlbumStarRating component 1`] = `
<span
  className="AlbumStarRating"
>
  <Star
    filled={false}
    key="0"
  />
  <Star
    filled={false}
    key="1"
  />
  <Star
    filled={false}
    key="2"
  />
  <Star
    filled={false}
    key="3"
  />
  <Star
    filled={false}
    key="4"
  />
  <Star
    filled={false}
    key="5"
  />
  <Star
    filled={false}
    key="6"
  />
  <Star
    filled={false}
    key="7"
  />
  <Star
    filled={false}
    key="8"
  />
  <Star
    filled={false}
    key="9"
  />
</span>
`;
